rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/util/locations/impl/ArrayLineOffsetMap.java

```diff
@@ -34,10 +34,12 @@
 public class ArrayLineOffsetMap implements LineColumnOffsetMap {
```

Comment:
changes here seem unrelated to the PR' topic? I'll try to find the commit message.

Reply:
I needed the inverse logic, since I have to find the cursor position in the ITree.

Reply:
yup, that is also used, but we needed to first translate form LSP utf16 offsets to rascal utf32 offsets.